src/three-components/shader-chunk/cube_uv_reflection_fragment.glsl.js

```diff
@@ -3,26 +3,23 @@ import {getFaceChunk, getUVChunk} from './common.glsl.js';
 export const cubeUVChunk = /* glsl */ `
 #ifdef ENVMAP_TYPE_CUBE_UV
 
-#define cubeUV_faceSize (256.0)
-#define cubeUV_maxMipLevel (8.0)
+const float cubeUV_maxMipLevel = 8.0;
+const float cubeUV_minMipLevel = 3.0;
```

Comment:
I almost wish these could be shared constants across this shader chunk and the PMREM generator (also `mipBase`). Or maybe uniforms? But that would be unnecessary since we can just "preprocess" them into the program source here..

Reply:
Yeah, really they should be uniforms, but since this is internal to three.js, we'll have to wait until it's upstreamed for that.